internal/mysqld/executor/executor.go

```diff
@@ -129,6 +140,242 @@ func (e *defaultExecutor) execSelect(ctx context.Context, n *planner.NodeSelectS
 	return wrapQueryResults(res), nil
 }
 
+func (e *defaultExecutor) execANNS(ctx context.Context, q *planner.NodeQuerySpecification, outputs []string) (*sqltypes.Result, error) {
+	if !q.Limit.IsSome() {
+		return nil, fmt.Errorf("limit not specified in the ANNS statement")
+	}
+	limit := q.Limit.Unwrap().Limit
+	offset := q.Limit.Unwrap().Offset
+
+	annsClause := q.Anns.Unwrap()
+
+	searchParams := map[string]string{
+		"anns_field": annsClause.Column.Name,
```

Comment:
make these three string literals  to be CONST.
currently . package proxy defines 
	AnnsFieldKey     = "anns_field"
	TopKKey          = "topk"
        OffsetKey        = "offset"
may be we can move these to common package ?